### ISSUE TITLE
Nuxt ESLint rules

### ISSUE THREAD

**Issue by luca-smartpricing** [nuxt-modules/prismic#213]
Hi!, 
could you insert some eslint rules for the new `@nuxt/eslint` module by default?  

E.g. this rule was essential:
```
{
  files: ['slices/**/*.vue'],
  rules: {
    'vue/multi-word-component-names': 'off'
  }
}
```

Thanks!

**Comment by lihbr** [comment]
Implemented in #214, should be released this week ☺️

**Comment by lihbr** [comment]
Released in 3.3.0 🎉
